### DISCUSSION TITLE
Make customizable whether magit-status-here favors staged or unstaged section

### DISCUSSION THREAD

**Discussion by lerouxrgd** [magit/magit#4814] [Ideas]
Hello, 

Currently `magit-status-here` sets `magit-status-goto-file-position` which makes `magit-status-setup-buffer` setup the status buffer as follows:

https://github.com/magit/magit/blob/36059e0b881cb1465cb5ad0099e55e00845a8222/lisp/magit-status.el#L436-L446

1.  if a relevant **staged** position exists, go to it
2.  if a relevant **unstaged** position exists, go to it

However it would be nice to be able to customize this flow and look for a relevant **unstaged** position first.

**Comment by wavexx** [comment]
I've hit this multiple times where I'm adjusting some changes to be staged and modifying them and I'd like to /always/ jump to the unstaged section, but found no quick way to do so.

**Reply by tarsius** [reply]
I agree that it makes more sense to favor going to the relevant unstaged changes. I'll modify it to behave like that (and then some), instead of making it customizable.

**Comment by tarsius** [comment]
Starting with 142e6330166614b191b7f94b765c90e74610b3f3 jumping to the untracked changes is favored. (The change was simple because I first [prepared](https://github.com/magit/magit/compare/731642756f...d81e6df443) to be simple.)
